devtools/conda-envs/test_min.yaml

```diff
@@ -1,17 +1,17 @@
-name: test_min
+name: test
 channels:
   - conda-forge
   - defaults
 dependencies:
   - python
   - pip
-  - sphinx
   - numpy==1.16.0
+  - sphinx
```

Comment:
this was just a switch around to make it match better with the order in `test_env.yaml`